syncer/main.py

```diff
@@ -1,51 +1,124 @@
 """
 A plugin to let two Gateways work together
 """
+import threading
+
 import six
 import time
 import requests
 import json
-from threading import Thread
 from plugins.base import om_expose, output_status, OMPluginBase, PluginConfigChecker, background_task
-import logging	
+from threading import Thread
```

Comment:
why import Thread if threading has been imported?

Reply:
Because we only need the Thread class from threading and nothing more

Reply:
Then you shouldn't import threading separately

Reply:
Done